com.unity.render-pipelines.high-definition/Editor/Material/Unlit/ShaderGraph/HDUnlitPass.template

```diff
@@ -48,7 +48,6 @@ Pass
     #if defined(_ENABLE_SHADOW_MATTE) && SHADERPASS == SHADERPASS_FORWARD_UNLIT
         #define LIGHTLOOP_DISABLE_TILE_AND_CLUSTER
         #define HAS_LIGHTLOOP
-        #define SHADOW_OPTIMIZE_REGISTER_USAGE 1
```

Comment:
Similarly to the question below, aren't shadow matte unlit with the shadow functions called? If so, why undefining SHADOW_OPTIMIZE_REGISTER_USAGE?

Reply:
double define, wasn't required here. part of clode cleanup